protocol.html

```diff
@@ -476,6 +480,14 @@ <h3 property="schema:name">Storage</h3>
 
                   <p>[<a href="https://github.com/solid/data-interoperability-panel/issues/10#issuecomment-598694029" rel="cito:citesAsSourceDocument">Source</a>] [<a href="https://github.com/solid/specification/issues/153#issuecomment-624630022" rel="cito:citesAsSourceDocument">Source</a>]</p>
 
+                  <p>Servers MUST keep track at least one <a data-link-type="dfn" href="#owner" id="ref-for-owner">owner</a> of a storage in an implementation defined way. The owner is first set at storage provisioning time and can be changed. The owner is allowed implicit control access on every resource in a storage.</p>
+
+                  <p>When a server wants to advertise the owner of a storage, the server MUST include the <code>Link</code> header with <code>rel="http://www.w3.org/ns/solid/terms#owner"</code> targeting the URI of the owner in the response of HTTP <code>HEAD</code> or <code>GET</code> requests targeting the root container.</p>
```

Comment:
```suggestion
                  <p>When a server wants to advertise the owner of a storage, the server MUST include the <code>Link</code> header with <code>rel="http://www.w3.org/ns/solid/terms#owner"</code> targeting the WebID of the owner in the response of HTTP <code>HEAD</code> or <code>GET</code> requests targeting the root container.</p>
```

Reply:
Almost interchangeable. I did consider WebID but decided to go with a URI-reference only. As it stands, there is no additional requirement (and to test) whether the reference must resolve to a document including the WebID. I'm not against using WebID, just think URI would be more appropriate here.